dashboard/app/graphs.go

```diff
@@ -187,6 +188,45 @@ func handleFoundBugsGraph(c context.Context, w http.ResponseWriter, r *http.Requ
 	return serveTemplate(w, "graph_histogram.html", data)
 }
 
+func handleCoverageGraph(c context.Context, w http.ResponseWriter, r *http.Request) error {
+	hdr, err := commonHeader(c, r, w, "")
+	if err != nil {
+		return err
+	}
+	yesterday := civil.DateOf(time.Now().Add(-1 * 24 * time.Hour))
+	monthAgo := yesterday.AddDays(-31)
+	hist, err := MergedCoverage(c, "upstream", monthAgo, yesterday)
```

Comment:
Should it be `hdr.Namespace` instead of `upstream`?

Reply:
Thanks. Done.